src/Downloader.ts

```diff
@@ -323,10 +329,14 @@ class Downloader {
 
     return articleRenderer.render({
       data,
+      redisStore,
```

Comment:
Any reason why the redisStore is not a singleton? Would be easier to handle?!

Reply:
Yes, Redis store will be handled more conveniently as a singleton. I did these changes in 231e403433a5a3d1a31370fe2ecf03a0fda317a0